action.yml

```
@@ -216,6 +228,8 @@ runs:
         BB_CACHE_LAYERS: ${{ inputs.use_cache }}
         BB_RETRY_PUSH_COUNT: ${{ inputs.retry_push_count }}
         BB_SQUASH: ${{ inputs.squash }}
+        BB_BUILD_CHUNKED_OCI: ${{ inputs.build_chunked_oci }}
+        BB_MAX_LAYERS: ${{ inputs.max_layers }}
```

```suggestion
        BB_BUILD_CHUNKED_OCI_MAX_LAYERS: ${{ inputs.max_layers }}
```
Based on the CLI changes